net/unbound/files/unbound.sh

```diff
@@ -1047,9 +1026,28 @@ unbound_uci() {
 
 ##############################################################################
 
+unbound_ismain() # true=0, false=1.
+{
+  if [ "$UNBOUND_N_RX_PORT" != "53" ] ; then
```

Comment:
There are a few use cases: (1) plain user with released LEDE/OpenWrt using LuCI software installer to add Unbound. (2) evaluation testing for power-users who want to see what Unbound offers. (3) this development testing. (4) ftp/samba into router with hack scripts. Bottom line is dnsmasq is default and Unbound is optional, so the scripts need to respect that.